src/utils/get-new-id.ts

```diff
@@ -0,0 +1,13 @@
+import {extractIdsFromConfig, generateUniqId, Config} from '../shared';
+
+type GetNewIdArgs = {
+    config: Config;
+    salt: Config['salt'];
+    counter: Config['counter'];
+    ids?: string[];
+};
+
+export function getNewId({config, salt, counter, ids = []}: GetNewIdArgs) {
```

Comment:
Let's add comment for args (or better in type GetNewIdArgs). It's not obvious that ids is used for kind of excluding (https://github.com/gravity-ui/dashkit/pull/41/files#diff-aab83ab4ef47bc5f82ffdc6f9b5c7bdea61325dba8344ea740b4aa01a2d28d3fR221) that provides uniqueness.

Reply:
maybe it's better rename to `excludeIds`